packages/docs/playwright/with-metamask.test.ts

```diff
@@ -68,48 +76,233 @@ describe(`Browser: ${browserType.name()} with Metamask`, () => {
         expect(await page.isVisible(`//*[text()='Current chain: ' and text()='1']`)).to.be.true
       })
 
-      let popupPromise = waitForPopup(context)
+      const popupPromise = waitForPopup(context)
       await page.click(XPath.text('button', 'Switch to Goerli'))
-      let popupPage = await popupPromise
+      const popupPage = await popupPromise
       await popupPage.click(XPath.text('button', 'Switch network'))
 
       await waitForExpect(async () => {
         expect(await page.isVisible(`//*[text()='Current chain: ' and text()='5']`)).to.be.true
       })
-
-      popupPromise = waitForPopup(context)
-      await page.click(XPath.text('button', 'Switch to Mainnet'))
-      popupPage = await popupPromise
-      await popupPage.click(XPath.text('button', 'Switch network'))
-
-      await waitForExpect(async () => {
-        expect(await page.isVisible(`//*[text()='Current chain: ' and text()='1']`)).to.be.true
-      })
     })
   })
 
   describe('Guides/Siwe', () => {
     it('Can sign in and sign out', async () => {
       await page.goto(`${baseUrl}Guides/Sign%20in%20with%20Ethereum`)
-
       await waitForExpect(async () => {
         expect(await page.isVisible(`//*[text()='Not logged in']`)).to.be.true
       })
-
       const popupPromise = waitForPopup(context)
       await page.click(XPath.text('button', 'Sign in'))
       const popupPage = await popupPromise
       await popupPage.click(XPath.text('button', 'Sign'))
-
       await waitForExpect(async () => {
         expect(await page.isVisible(`//*[text()='Logged in with ']`)).to.be.true
       })
-
       await page.click(XPath.text('button', 'Sign out'))
-
       await waitForExpect(async () => {
         expect(await page.isVisible(`//*[text()='Not logged in']`)).to.be.true
       })
     })
   })
 })
+
+if (process.env.CI) {
```

Comment:
How about, instead of this, having:

```
describe(`Browser: ${browserType.name()} with Gnosis Safe`, function () {
  if (!process.env.DOCS_GNOSIS_OWNER_FIRST) this.skip()
```

(If that works).

So that we can see the tests which are skipped.